pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[project]
+name = "itscalledsoccer"
+description = "A Python wrapper around the unofficial FotMob API"
+readme = "README.md"
+license = { text = "MIT" }
+requires-python = ">=3.8"
+authors = [
+  { name = "American Soccer Analysis", email = "americansocceranalysis@gmail.com" },
+]
+keywords = [
+    "stats",
+    "soccer",
+    "api",
+    "football",
+    "american",
+    "machine learning"
+]
+classifiers = [
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11"
+]
+dependencies = [
+    "requests==2.31.0",
+    "CacheControl==0.12.11",
+    "rapidfuzz==3.0.0",
+    "pandas==2.0.1"
```

Comment:
Looks like you went backward on the versions of `CacheControl` and `pandas`. Not sure if that was intentional.

Reply:
Nope, good catch